File: SyncKit/Classes/CoreData/QSRecord+CoreDataProperties.m
//
//  QSRecord+CoreDataProperties.m
//  
//
//  Created by Manuel Entrena on 22/01/2018.
//
//

#import "QSRecord+CoreDataProperties.h"

@implementation QSRecord (CoreDataProperties)

+ (NSFetchRequest<QSRecord *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"QSRecord"];
}

@dynamic encodedRecord;
@dynamic forEntity;

@end

File: SyncKit/Classes/CoreData/QSRecord+CoreDataProperties.swift
//
//  QSRecord+CoreDataProperties.swift
//  
//
//  Created by Manuel Entrena on 02/06/2019.
//
//

import Foundation
import CoreData


extension QSRecord {

    @nonobjc class func fetchRequest() -> NSFetchRequest<QSRecord> {
        return NSFetchRequest<QSRecord>(entityName: "QSRecord")
    }

    @NSManaged var encodedRecord: NSData?
    @NSManaged var forEntity: QSSyncedEntity?

}
